x/denommetadata/types/params.go

```diff
@@ -0,0 +1,68 @@
+package types
+
+import (
+	"fmt"
+
+	errorsmod "cosmossdk.io/errors"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+var (
+	KeyAllowedAddresses     = []byte("AllowedAddresses")
+	DefaultAllowedAddresses = []string(nil) // no one allowed
+)
+
+// ParamTable for minting module.
```

Comment:
invalid godoc

Reply:
Just realized other place have this error as well lol